prisma/schema.prisma

```diff
@@ -7,27 +7,32 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
-model Post {
-  id        Int      @id @default(autoincrement())
-  title     String   @db.VarChar(255)
-  createdAt DateTime @default(now()) @db.Timestamp(6)
-  content   String?
-  published Boolean  @default(false)
-  authorId  Int
-  User      User     @relation(fields: [authorId], references: [id])
+model User {
+  id         Int         @id @default(autoincrement())
+  username   String
+  hash       String
+  chapters   Chapter[]
```

Comment:
This relation should be one-to-one. The `User` model contains user credential so each `User` must belong to one `Chapter`/`Recipient`/`Admin` and vice-versa.

More info [here](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/one-to-one-relations)